docs/platforms-release-process.md

```diff
@@ -261,6 +262,10 @@ Then ensure commits look okay on both branches
 
 Once all the repos are branched, we focus on testing & fixing all of the regressions we find.
 
+### Prerequisite: npm test
+
+Ensure that `npm test` succeeds. This can normally be done by checking for a green build.
```

Comment:
Hm, don't we want the developer to _also_ run the tests locally?

Reply:
Reworded, got rid of "normally", and added info that we are waiting for <https://github.com/apache/cordova-osx/issues/71> to be resolved.

I generally run unit tests on my local machine. To run the full `npm test` suite often takes much longer on my local machine than on Travis CI. I think having a green build on CI is much more important than verifying it locally.